src/v/kafka/protocol/schemata/generator.py

```diff
@@ -1075,8 +1075,8 @@ class response;
 {%- else %}
 {{ fname }} = reader.read_array([version](request_reader& reader) {
```

Comment:
It kinda looks like it's used for `kafka::produce_request_record_data`, perhaps that's dead code.

Reply:
hmm, i'm grepping for `produce_request_record_data` here and i'm not seeing anything. should i be looking at something else?

Reply:
chatted with Rob about this who wrote this part of the generator and got a :+1: